docs/client/rest/rest_api.md

```diff
@@ -261,6 +261,71 @@ Create an operation with GET_SCHEMAS type
 |:-----------|:----------------------------|:-------|
 | identifier | The identifier of operation | String |
 
+## Operation Resource
+
+### GET /operations/${operationHandle}/event
+
+Get the current event of the operation by the specified operation handle.
+
+#### Response Body
+
+The [KyuubiOperationEvent](#kyuubioperationevent).
+
+### PUT /operations/${operationHandle}
+
+Put an action to the operation
+
+#### Request Body
+
+| Name   | Description                                                                 | Type   |
+|:-------|:----------------------------------------------------------------------------|:-------|
+| action | The action that is applied the operation, for example: 'cancel' and 'close' | String |
```

Comment:
Let's clarify the action description a bit like the above comments.

Reply:
The preview results.
![image](https://user-images.githubusercontent.com/12015546/199874068-63b9f377-c369-40aa-971c-df37cf3cdb9d.png)